Application/Telegram/MainActions/NewBookingCommand.cs

```diff
@@ -91,10 +91,10 @@ public static async Task HandleAsync(Update update, TelegramBotClient telegraBot
                 case UserState.NewBookingIsSelectedSelectingEndDateMonth:
                     DateHelper.EndMonthUpdater(update.CallbackQuery, ref user);
                     await new SendDayCommand(mediator, telegraBotClient).SendAsync(
-                        update.CallbackQuery
-                        ,"Please Select Workplace Booking Type "
-                        , "BACKNewBookingIsSelected"
-                        ,"End");
+                        update.CallbackQuery,
+                        "Please select end date day",
```

Comment:
mb it's better to leave only "select end date" (not critical)

Reply:
done